content/applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst

```diff
@@ -0,0 +1,150 @@
+:guilabel:`New` to create a new |MO|.
+
+On the new |MO|, select the product to be produced from the :guilabel:`Product` drop-down menu. The
+:guilabel:`Bill of Material` field auto-populates with the associated Bill of Materials (BoM).
+
+If a product has more than one |BOM| configured for it, the specific |BOM| can be selected in the
+:guilabel:`Bill of Material` field, and the :guilabel:`Product` field auto-populates with the
+associated product.
+
+After a |BOM| has been selected, the :guilabel:`Components` and :guilabel:`Work Orders` tabs
+auto-populate with the components and operations specified on the |BOM|. If additional components or
+operations are required for the |MO| being configured, add them to the :guilabel:`Components` and
+:guilabel:`Work Orders` tabs by clicking :guilabel:`Add a line`.
+
+Finally, click :guilabel:`Confirm` to confirm the |MO|.
+
+Process pick components transfer
+================================
+
+After confirming a three-step |MO|, a :guilabel:`Transfers` smart button appears at the top of the
+page. Click it to be taken to the :guilabel:`Transfers` page for the |MO|. The page lists two
+transfers: *WH/PC/XXXXX* (the pick components transfer), and *WH/SFP/XXXXX* (the store finished
+products transfer).
+
+Select :guilabel:`WH/PC/XXXXX` to open the pick components transfer for the |MO|. This transfer is
+used to track the movement of components from the locations where they are stored to the location
+where they are used to manufacture the product.
+
+After transferring the components out of their storage location, click :guilabel:`Validate` at the
+top of the transfer, followed by :guilabel:`Apply` on the :guilabel:`Immediate Transfer?` pop-up
+window that appears. Doing so marks the transfer as :guilabel:`Done`, and updates inventory counts
+to reflect the quantity of components transferred.
+
+Finally, return to the |MO| by clicking the :guilabel:`WH/MO/XXXXX` breadcrumb at the top of the
+page.
+
+.. image:: three_step_manufacturing/mo-bread-crumb.png
+   :align: center
+   :alt: The manufacturing order bread crumb on a pick components transfer.
+
+Process manufacturing order
+===========================
+
+An |MO| is processed by completing all of the work orders listed under its :guilabel:`Work Orders`
+tab. This can be done on the |MO| itself, or from the work order tablet view.
+
+Basic workflow
+--------------
+
+To complete work orders from the |MO| itself, begin by navigating to :menuselection:`Manufacturing
+--> Operations --> Manufacturing Orders`, and then select a manufacturing order.
+
+On the |MO| page, select the :guilabel:`Work Orders` tab. Once work begins on the first work order
+that needs to be completed, click the :guilabel:`Start` button for that work order. Odoo
+*Manufacturing* then starts a timer that keeps track of how long the work order takes to complete.
+
+.. image:: three_step_manufacturing/start-button.png
```

Comment:
this image doesn't *appear* to be compressed I was able to get it down to about ~14kb